packages/radio-button/package.json

```diff
@@ -13,13 +13,14 @@
   },
   "main": "build/index.js",
   "module": "build/index.es.js",
-  "style": "build/index.css",
   "files": [
     "build/*"
   ],
   "dependencies": {
+    "@hig/behaviors": "^1.1.1",
+    "@hig/theme-context": "^1.0.1",
     "@hig/utils": "^0.3.0",
-    "classnames": "^2.2.5",
+    "emotion": "^10.0.6",
```

Comment:
I think for consistency it might be better to keep `emotion` at `10.0.0` like the other components are currently using.